From 19e1695794ad1815c6b5f696c484c66354d4cafe Mon Sep 17 00:00:00 2001
From: huweiguo <hwg@rock-chips.com>
Date: Thu, 13 Apr 2017 17:04:33 +0800
Subject: [PATCH] arm64: rockchip_defconfig: enable TCP SYN_COOKIES

solve some vst video apk can't play video with kmsg:
"TCP: request_sock_TCP: Possible SYN flooding on port 9000. Dropping request.  Check SNMP counters."

Change-Id: I4fc94307195464c8e8180217eccb9a590eb3c5b6
Signed-off-by: huweiguo <hwg@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 4b999d1abc85..2a4a3190c90f 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -102,6 +102,7 @@ CONFIG_IP_MULTIPLE_TABLES=y
 CONFIG_IP_ROUTE_VERBOSE=y
 CONFIG_IP_PNP=y
 CONFIG_IP_PNP_DHCP=y
+CONFIG_SYN_COOKIES=y
 CONFIG_INET_AH=y
 CONFIG_INET_ESP=y
 CONFIG_INET_IPCOMP=y
-- 
2.35.3

